Dockerfile

```diff
@@ -1,27 +1,53 @@
-FROM fedora:latest
+# Arguments for DEV's (comment static FROM and uncomnnet #DEV ones)
+ARG UBI_VERSION=9
+ARG OPENJDK_VERSION=17
+ARG UBI_BUILD_TAG=latest
+ARG UBI_RUNTIME_TAG=latest
+ARG IMAGE_BUILD=registry.access.redhat.com/ubi${UBI_VERSION}/openjdk-${OPENJDK_VERSION}:${UBI_TAG}
```

Comment:
does this run the image build repeatedly for each arch, incl downloading maven artifacts? is it possible to improve on this in future? either cache artifacts, or do one build for all the multiarch images?

Reply:
Yes, there is possibility to use GitHub Action Cache for Maven and use it in every job. Then in the image we can reuse it in builder stage and take cache from the path  `COPY . /app`

Reply:
💌